content/en/core/overview/pwa.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Installation as a Progressive Web App"
+linkTitle: "Progressive Web App"
+weight: 2
+description: >
+  What does it mean that the CHT Core web application is a Progressive Web App.
+---
+
+## What is a Progressive Web App (PWA)?
+
+A PWA is a web application that can be used like a website in the browser, but the user can choose to "install" it. This means a shortcut is added to the home screen of the device, and when the application is run it doesn't have the usual browser address bar and tabs so it looks like a regular application.
+
+The CHT Core webapp has been developed to be a PWA to give users more choice about how applications are installed.
+
+## Benefits of PWA installation
+
+Using a PWA avoids the need to build an Android application and publish it to the Play Store which can be cumbersome and slow. This also means there's no need for the user to install or update the Android application.
```

Comment:
> This also means there's no need for the user to install or update the Android application.

It might be worth noting how the PWA is "updated".  Something like "The PWA is updated automatically when syncing with the CHT server."

(I know this is not really the same "update" as the Android application, but I think a reader who is not super familiar with PWA's still might wonder how things stay up do date if there is no app to update...)

Reply:
Good call. Added.